### PR TITLE
feat(contact-info): NO-JIRA add `avatarColor` prop to contact info recipe

### PR DESCRIPTION
**Pull request by TehuelDialpad**
# adds an `avatarColor` prop to contact info recipe

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Description

On this PR I add a `avatarColor` prop to `contactInfo` recipe component to be passed to `avatar` internal component.

## :bulb: Context

This new prop is needed as part of an initiative to display some contacts with a grey background in the avatar. With this PR the documentation (Storybook) for the recipe is also updated.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [x] I have added / updated unit tests.
- [x] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)
- [ ] I have validated components with a screen reader.
- [ ] I have validated components keyboard navigation.

For all CSS changes:

- [ ] I have used design tokens whenever possible.
- [ ] I have considered how this change will behave on different screen sizes.
- [ ] I have visually validated my change in light and dark mode.
- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.

If new component:

- I am exporting any new components or constants:
  - [ ] from the index.js in the component directory.
  - [ ] from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
- [ ] I have added the styles for the new component to the `packages/dialtone-css` package.
- [ ] I have created a page for the new component on the documentation site in `apps/dialtone-documentation`.
- [ ] I have added the new component to `common/components_list.cjs`
- [ ] I have created a component story in storybook
- [ ] I have created story / stories for any relevant component variants in storybook
- [ ] I have created a docs page for the component in storybook.
- [ ] I have checked that changing all props/slots via the UI in storybook works as expected.

### CODE REVIEW

File: packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.test.js [hunk 1]
```diff
@@ -115,6 +115,22 @@ describe('DtRecipeContactInfo Tests', () => {
       });
     });
 
+    describe('When `avatarColor` is defined', () => {
+      beforeEach(async () => {
+        await wrapper.setProps({
+          avatarColor: '000',
+        });
+        await wrapper.vm.$nextTick();
```
Comment (juliodialpad) [inline packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.test.js]: This shouldn't be needed, please test if it works after removing it.

File: packages/dialtone-vue3/recipes/item_layout/contact_info/contact_info.test.js [hunk 2]
```diff
@@ -108,6 +108,22 @@ describe('DtRecipeContactInfo Tests', () => {
       });
     });
 
+    describe('When `avatarColor` is defined', () => {
+      beforeEach(async () => {
+        await wrapper.setProps({
+          avatarColor: '000',
+        });
+        await wrapper.vm.$nextTick();
```
Comment (juliodialpad) [inline packages/dialtone-vue3/recipes/item_layout/contact_info/contact_info.test.js]: Same here, please remove and test

**Review by juliodialpad — APPROVED** [review]
Looks good, just apply the requested changes before merging please.

**Review by braddialpad — APPROVED** [review]

File: packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.test.js (hunk 1 above)
Comment (TehuelDialpad) [reply packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.test.js]: The set of `avatarColor ` value is needed. The next part can be deleted and the test still works as expected. Thanks for the comment.

**Pull request merged**